src/main/java/teammates/common/datatransfer/attributes/StudentAttributes.java

```diff
@@ -175,25 +175,23 @@ public boolean isEnrollInfoSameAs(StudentAttributes otherStudent) {
         // id is allowed to be null when the student is not registered
         Assumption.assertNotNull(team);
         Assumption.assertNotNull(comments);
-
```

Comment:
Don't remove this whitespace

Reply:
@amrut-prabhu Thanks for the review, I will surely correct them all.